pkg/ingress/query.go

```diff
@@ -150,3 +158,41 @@ func getRule(host string, path string, pathType *networkingv1.PathType, backend
 	}
 	return rule
 }
+
+func getIngressClass() (string, bool) {
+	ingressClass := ""
+	nginxIngressAvailable := false
+	// This statement only work if it is running inside cluster, if you want to test outside replace this statement with
+	// config, err := clientcmd.BuildConfigFromFlags("", "PATH_TO_YOUR_CLUSTER_CONFIG_FILE")
```

Comment:
This comment is not needed because it doesn't make sense to run this application outside a cluster of without being connected to one.